habitTrackerApp.py

```diff
@@ -275,45 +277,76 @@ class MyBoxLayout(BoxLayout):
     screenDict[date.today().strftime('%Y-%m')] = initialScreen
     theEverywhereDate = ''
     colorDict = {
-            "red": [(1,0,0,1),'Push Ups'],
-            "green": [(0,1,0,1),'Get Milk'],
-            "blue": [(0,0,1,1),'Sing Song'],
-            "3": [(1,1,0,1),'Sit Ups'],
-            "4": [(0,1,1,1),'Sleeping'],
+            "blue": [(138/255,166/255,1,1),'Push Ups'],
+            "green": [(172/255,1,181/255,1),'Get Milk'],
+            "red": [(1,166/255,136/255,1),'Sing Song'],
+            "pink": [(1,174/255,236/255,1),'Sit Ups'],
+            "orange": [(1,204/255,139/255,1),'Sleeping'],
             "5": [(1,0,1,1),'Jumping'],
             "white": [(1,1,1,1),'Stretching'],
             "gridBlue": [(0.082,0.298,1,0.6),'Eat Vegetables']
             }
     buttonList = []
     TheTodayButton = ''
-    currentSideButton = ObjectProperty()
+    currentSideButton = ''
```

Comment:
it's is better to use None rather than '' which has a different meaning. 
'' means we expect this object to be a string but your intent is that this is just uninitialized.